### ISSUE TITLE
[Feature Request: Alerting] Accept feedback on quality of alert

### ISSUE THREAD

**Issue by mattttt** [grafana/grafana#7100]
Been thinking a lot lately about how we can crowdsource feedback on the quality of the alerts being set up. Knowing that alert fatigue is very real, we need a fast way for teams to know what alerts are being actively ignored. 

It feels that we can put together a very simple implementation using some basic slack integration, and then expand the integration to include other methods of reporting. I'm envisioning something like this to accompany alerts in slack:

![image](https://cloud.githubusercontent.com/assets/2886187/21571032/5c57359e-ce99-11e6-96fb-eb8777f79980.png)

Ref: https://chatbotsmagazine.com/better-slack-app-ux-with-buttons-23a65b827f46

Then on the Grafana side, we'd need a way to keep track of the feedback on a per alert basis. Once it crosses some threshold, we can alert admins to the questionable quality of the alert, show an icon on the alert list, and perhaps even include a warning on future alerts ("The team thinks this alert is unreliable").

**Comment by torkelo** [comment]
That is a pretty interesting idea. Then we could have counters for valid/false and show that somewhere.

**Comment by mattttt** [comment]
Yes, totally! And assuming it's justing hitting the API, people can extend it in all sorts of interesting ways.
